libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -76,11 +85,18 @@ interface ScannerStateUnchangedByExport {
   pollsState?: PollsState;
 }
 
-interface ExportOptions {
+interface CentralScannerOptions {
+  scannerType: 'central';
+}
+
+interface PrecinctScannerOptions {
+  scannerType: 'precinct';
   arePollsClosing?: boolean;
   isFullExport?: boolean;
 }
 
+type ExportOptions = CentralScannerOptions | PrecinctScannerOptions;
```

Comment:
There are ways to get by without a `scannerType` field in the options, but I ultimately found that the `scannerType` field made the code clearer